actors/builtin/miner/miner_test.go

```diff
@@ -3582,11 +3867,17 @@ func newHarness(t testing.TB, provingPeriodOffset abi.ChainEpoch) *actorHarness
 	rwd := big.Mul(big.NewIntUnsigned(10), big.NewIntUnsigned(1e18))
 	pwr := abi.NewStoragePower(1 << 50)
 	h := &actorHarness{
-		t:        t,
-		receiver: receiver,
-		owner:    owner,
-		worker:   worker,
-		key:      workerKey,
+		t:           t,
+		receiver:    receiver,
+		owner:       owner,
+		worker:      worker,
+		key:         workerKey,
+		beneficiary: owner,
+		beneficiaryInfo: miner.BeneficiaryInfo{
+			Quota:      abi.TokenAmount{},
+			Expiration: 0,
+			UsedQuota:  abi.TokenAmount{},
+		},
```

Comment:
I see you need the beneficiary address in the harness in order to set mock expectations, but I don't think you need the beneficiaryInfo.